app/assets/javascripts/qpixel_api.js

```diff
@@ -12,15 +12,42 @@ window.QPixel = {
     return token;
   },
 
+  /** Counts notifications popped up at any time. */
+  var popped_modals_ct = 0;
```

Comment:
We're using ES6, so `const` and `let` - `const` if the value won't change, `let` if you need to re-assign to the variable later.